src/Analyzer.Utilities/Extensions/ISymbolExtensions.cs

```diff
@@ -151,6 +161,30 @@ public static bool MatchFieldByName(this ISymbol member, INamedTypeSymbol type,
             return member != null && member.Kind == SymbolKind.Field && member.MatchMemberByName(type, name);
         }
 
+        // Define the format in for displaying member names. The format is chosen to be consistent
+        // consistent with FxCop's display format.
+        private static readonly SymbolDisplayFormat s_memberDisplayFormat =
+            // This format omits the namespace.
+            SymbolDisplayFormat.CSharpShortErrorMessageFormat
+                // Turn off the EscapeKeywordIdentifiers flag (which is on by default), so that
+                // a method named "@for" is displayed as "for".
+                // Turn on the UseSpecialTypes flat (which is off by default), so that parameter
+                // names of "special" types such as Int32 are displayed as their language alias,
+                // such as int for C# and Integer for VB.
+                .WithMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
+
+        /// <summary>
+        /// Format member names in a way consistent with FxCop's display format.
+        /// </summary>
+        /// <param name="member"></param>
+        /// <returns>
+        /// A string representing the name of the member in a format consistent with FxCop.
+        /// </returns>
+        public static string FormatMemberName(this ISymbol member)
```

Comment:
There are only four calls to this API, all in the context of `ReportDiagnostic`. But in any case, this method does not emit the namespace, it only emits the type name. I have tests for this, for example:

```
        // The following tests are just to ensure that the messages are formatted properly
        // for types within namespaces.
        [Fact]
        public void CA1812_CSharp_Diagnostic_UninstantiatedInternalClassInNamespace()
        {
            VerifyCSharp(
@"namespace N
{
    internal class C { }
}",
                GetCSharpResultAt(3, 20, AvoidUninstantiatedInternalClassesAnalyzer.Rule, "C"));
        }
```

Is this not what we want?